apps/system/test_softlockup/Kconfig

```diff
@@ -0,0 +1,12 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config SYSTEM_SOFTLOCKUP_TEST
```

Comment:
Is this a system module? not example?

Reply:
@sunghan-chang ,  It's a testcase to simulate the lockup and verify the system softlockup feature. If you are not convinced, we can discuss offline for better config name

Reply:
Because this is a sample code to test your functionallity, I think it should be in example, not system.